src/app/utils/plug-descriptions.ts

```diff
@@ -85,29 +85,9 @@ export function usePlugDescriptions(
   const perks = getPerkDescriptions(plug, defs, statAndBungieDescStrings);
 
   if (showCommunityDescription && allClarityDescriptions) {
-    let clarityPerk = allClarityDescriptions[plug.hash];
-
-    // if we couldn't find a Clarity description for this perk, fall back to the non-enhanced perk variant
-    if (!clarityPerk) {
-      const regularPerkHash = enhancedPerkToRegularPerk[plug.hash];
-      if (regularPerkHash) {
-        clarityPerk = allClarityDescriptions[regularPerkHash];
-      }
-    }
-
-    if (clarityPerk && !clarityPerk.statOnly) {
-      // strip out any strings that are used in the Bungie description
-      const communityInsightWithoutBungieStrings = stripUsedStrings(
-        clarityPerk,
-        statAndBungieDescStrings
-      );
-      if (communityInsightWithoutBungieStrings) {
-        // if our stripped community description is truthy, we know it contains at least 1 unique string
-        // we only want to strip out Bungie description strings if we're also showing the Bungie description
-        result.communityInsight = showBungieDescription
-          ? communityInsightWithoutBungieStrings
-          : stripUsedStrings(clarityPerk, statStrings);
-      }
+    const clarityPerk = allClarityDescriptions[plug.hash];
```

Comment:
```suggestion
    let clarityPerk = allClarityDescriptions[plug.hash];

    // if we couldn't find a Clarity description for this perk, fall back to the non-enhanced perk variant
    if (!clarityPerk) {
      const regularPerkHash = enhancedPerkToRegularPerk[plug.hash];
      if (regularPerkHash) {
        clarityPerk = allClarityDescriptions[regularPerkHash];
      }
    }
```

Reinstating this logic would allow DIM to fall back to displaying the regular perk's community description if a community description doesn't exist for the enhanced perk yet. I understand that Clarity has a process in place to catch these instances, but this is still likely to happen when an enhanced version of an existing trait is newly added.

For example, with Deep Stone Crypt weapons becoming craftable next season, we'd expect an Enhanced Reconstruction perk, but Clarity doesn't have a community description associated with the enhanced version of this perk. Rather than have no community description shown on the enhanced perk, it would be more useful to display the regular perk's community description. Once Clarity adds a description for the enhanced perk, DIM will correctly pick up the new description.

Reply:
Well it's up to you if you want it I can stay 
Because I removed this and we had this conversation I found and fixed a bug and added an automated system which will help us not only with enhanced perks but also some other ones

Reply:
Yep, my preference is we keep this logic, better to be safe than sorry. Great to hear that it motivated some improvements upstream!